### PR TITLE
Configure istio to use tide

### PR DESCRIPTION
**Pull request by jwendell**
And set the context options so that it respect the 2 required approvals

### CODE REVIEW

File: prow/config.gen.yaml [hunk 1]
```diff
@@ -1271,6 +1271,8 @@ tide:
   merge_method:
     maistra: squash
   target_url: https://prow.maistra.io/tide
+  context_options:
+    from-branch-protection: true
```
Comment (jwendell) [inline prow/config.gen.yaml:1275]: This seems to be causing failures...

**Comment by brian-avery** [comment]
Most of what I'm seeing here is: 

"required prow jobs require branch protection" and "Error getting branch protection for....". 

Looking at https://github.com/kubernetes/test-infra/blob/master/prow/config/branch_protection.go#L269,

it looks like this is created when `protect` is configured to false

**Comment by brian-avery** [comment]
Now seeing: 

the following orgs or repos enable the lgtm plugin but do not require the lgtm label for merging: [repo: maistra/istio]

From the prow docs (https://prow.k8s.io/plugins):

The lgtm plugin manages the application and removal of the 'lgtm' (Looks Good To Me) label which is typically used to gate merging.

I don't think we need the lgtm plugin as we use ok-to-merge. Disabling for now. @dgn @jwendell please let me know if I'm incorrect.

**Comment by brian-avery** [comment]
@jwendell ptal

**Review by dgn — APPROVED** [review]
lgtm. will you make sure to deactivate mergify once this is ready?

**Comment by brian-avery** [comment]
Yeah, I can do that.

**Pull request merged**